ansible/configs/cloud-credentials/post_software.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Step 005 Post Software
+  hosts: localhost
+  become: false
+  gather_facts: false
+  tags:
+    - step005
+    - post_software
+  tasks:
+    - debug:
+      msg: "Step 005 Post Software"
+
+    - name: Add AWS sandbox credentials if enabled
+      when:
+      - cloud_provider == 'ec2'
+      block:
+      - name: Set facts for AWS credentials
+        set_fact:
+          student_access_key_id: "{{ cloudformation_out_final.stack_outputs.StudentUserAccessKey }}"
+          student_secret_access_key: "{{ cloudformation_out_final.stack_outputs.StudentUserSecretAccessKey }}"
```

Comment:
I would find it more logical to have this portion of code in the `post_infra.yml` playbook